build/spas.ts

```diff
@@ -150,11 +150,12 @@ export async function buildSPAs(options) {
         { prefix: "about", pageTitle: "About MDN" },
         { prefix: "community", pageTitle: "Contribute to MDN" },
       ];
+      const urlLocale = VALID_LOCALES.get(locale) || locale;
```

Comment:
This is bad. The double rendering was solving issues where we SSRed with `en-us` and the re-rendering after hydration fixed that. 🧨